<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Fin_Deposit</class>
 <widget class="QWidget" name="Fin_Deposit">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>531</width>
    <height>781</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>531</width>
    <height>781</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>531</width>
    <height>781</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="Clear">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>110</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
    border-radius:  10px;
    background-color:  rgb(255,140, 0);
    color:  rgb(255, 255, 255);
    font-size:  15px;
}

QPushButton:pressed {
    background-color:  rgb(118, 118, 118);
}</string>
   </property>
   <property name="text">
    <string>Clear</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Calculate">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>110</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
    border-radius:  10px;
    background-color:  rgb(255,140, 0);
    color:  rgb(255, 255, 255);
    font-size:  15px;
}

QPushButton:pressed {
    background-color:  rgb(118, 118, 118);
}</string>
   </property>
   <property name="text">
    <string>Calculate</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_rate">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>226</y>
     <width>173</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="Rate">
   <property name="geometry">
    <rect>
     <x>124</x>
     <y>226</y>
     <width>91</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Interest rate</string>
   </property>
  </widget>
  <widget class="QLabel" name="Payout">
   <property name="geometry">
    <rect>
     <x>124</x>
     <y>295</y>
     <width>48</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Payout</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_dep">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>160</y>
     <width>173</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="Replenishment">
   <property name="geometry">
    <rect>
     <x>124</x>
     <y>328</y>
     <width>103</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Replenishment</string>
   </property>
  </widget>
  <widget class="QComboBox" name="box_payout">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>295</y>
     <width>173</width>
     <height>25</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Monthly</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Quarterly</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Annually</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>At the end of the term</string>
    </property>
   </item>
  </widget>
  <widget class="QComboBox" name="box_capitalisation">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>438</y>
     <width>173</width>
     <height>25</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>No</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Yes</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="Tax_kind">
   <property name="geometry">
    <rect>
     <x>124</x>
     <y>262</y>
     <width>56</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Tax rate</string>
   </property>
  </widget>
  <widget class="QComboBox" name="box_replenishment">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>328</y>
     <width>173</width>
     <height>25</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>No</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Yes</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="Capitalization">
   <property name="geometry">
    <rect>
     <x>124</x>
     <y>438</y>
     <width>94</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Capitalisation</string>
   </property>
  </widget>
  <widget class="QLabel" name="Period">
   <property name="geometry">
    <rect>
     <x>124</x>
     <y>193</y>
     <width>102</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Term</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_tax_kind">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>260</y>
     <width>173</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_period">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>193</y>
     <width>173</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="Deposit_amount">
   <property name="geometry">
    <rect>
     <x>124</x>
     <y>160</y>
     <width>110</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Deposit amount</string>
   </property>
  </widget>
  <widget class="QLabel" name="rate">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>569</y>
     <width>105</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Amount of rate</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_am_rate">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>568</y>
     <width>156</width>
     <height>25</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="tax">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>600</y>
     <width>98</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Amount of tax</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_am_tax">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>599</y>
     <width>156</width>
     <height>25</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="money">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>538</y>
     <width>127</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Amount of money </string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_am_money">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>538</y>
     <width>156</width>
     <height>25</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>468</y>
     <width>350</width>
     <height>25</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QRadioButton" name="radioButton_4">
      <property name="text">
       <string>Montly</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="radioButton_5">
      <property name="text">
       <string>Quarterly</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="radioButton_6">
      <property name="text">
       <string>Annually</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>360</y>
     <width>350</width>
     <height>25</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QRadioButton" name="radioButton">
      <property name="text">
       <string>Montly</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="radioButton_2">
      <property name="text">
       <string>Quarterly</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="radioButton_3">
      <property name="text">
       <string>Annually</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLineEdit" name="lineEdit_repl">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>400</y>
     <width>173</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
